model-server/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    kotlin("jvm")
+    application
+}
+
+val modelix_version: String by project
+
+dependencies {
+    implementation("org.modelix:model-server-fatjar:$modelix_version")
+}
+
+application {
+    mainClass.set("org.modelix.model.server.Main")
+    applicationDefaultJvmArgs = listOf("-XX:MaxRAMPercentage=85")
+
+    // well this would be nice. if it worked...
```

Comment:
Why doesn't this work?

Reply:
only JVM args are handled:
https://github.com/gradle/gradle/blob/master/subprojects/plugins/src/main/java/org/gradle/api/plugins/ApplicationPlugin.java#L160

Reply:
```suggestion
    // well this would be nice if it was actually supported by the gradle plugin...
```